ide-configurations/netbeans/README.md

```diff
@@ -0,0 +1 @@
+TODO Find a contributor
```

Comment:
:grin: I guess @olukas will think about the contribution once he is done cleaning up his mailbox. :fire: